app/controllers/recipe_foods_controller.rb

```diff
@@ -0,0 +1,75 @@
+class RecipeFoodsController < ApplicationController
+  before_action :set_recipe_food, only: %i[show edit update destroy]
+
+  # GET /recipe_foods or /recipe_foods.json
+  def index
+    @recipe_foods = RecipeFood.all
+  end
+
```

Comment:
- [ ] Kindly fix the N + 1 query issue here. We need to fetch recipes and foods at the same time without hitting the database every time.